src/webportal/src/app/job/job-view/fabric/tachyons.css

```diff
@@ -0,0 +1,2152 @@
+/* TACHYONS v0.0.1 | https://github.com/tachyons-css/tachyons */
```

Comment:
Have you modified this file? Can we add it as a dependence?

Reply:
Yes. It's a tachyons css without normalize.css generated by tachyons generator.